### PR TITLE
Approved tags and usecases and clean up existing metadata

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/30007

## Description
- Added approved tags and usecases files
- Cleaned non approved usecases and tags

### CODE REVIEW

File: Tests/Marketplace/approved_tags.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+[
```
Comment (yaakovi) [inline Tests/Marketplace/approved_tags.json:1]: shouldn't we have an `IT` tag too?

File: Tests/Marketplace/approved_tags.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"Alerts",
+	"Attack",
+	"Breach",
+	"COVID-19",
+	"Compliance",
+	"Crisis",
+	"Dark Web",
+	"Email",
+	"Failed Incidents",
+	"Failed Integrations",
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:11]: I don't think these are valid tags 
@michalgold

Comment (anara123) [reply Tests/Marketplace/approved_tags.json:11]: "Failed Incidents"
"Failed Integrations"

File: Tests/Marketplace/approved_tags.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	"Forensics",
+	"GDPR",
+	"HIPAA",
+	"IAM",
+	"Incident Handling",
+	"Incident Response",
+	"IoT",
+	"Machine Learning",
+	"Malware",
+	"Mirroring",
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:21]: Mirroring - don't think is valid tag

File: Tests/Marketplace/approved_tags.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	"Network",
+	"PII",
+	"Pandemic",
+	"Pcap",
+	"Risk",
+	"Rules",
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:27]: same here - what Risk and Rules means?

File: Tests/Marketplace/approved_tags.json (hunk 1 above)
Comment (anara123) [reply Tests/Marketplace/approved_tags.json:1]: definetly

File: Tests/Marketplace/approved_tags.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	"Scam",
+	"Security",
+	"Security Analytics",
+	"Simulation",
+	"Spam",
+	"TIM",
+	"Threat Intelligence",
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:34]: we can add Utilities

File: Tests/Marketplace/approved_usecases.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	"Compliance",
+	"Crisis Management",
+	"Default",
+	"Email Communication",
+	"Employee Offboarding",
+	"GDPR Breach Notification",
+	"HIPAA Breach Notification",
+	"Health Check",
+	"Hunting",
+	"Identity And Access Management",
```
Comment (anara123) [inline Tests/Marketplace/approved_usecases.json:16]: "Identity And Access Management" or "Identity Lifecycle Management"?
@glicht @michalgold

File: Tests/Marketplace/approved_tags.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+	"Pandemic",
+	"Pcap",
+	"Risk",
+	"Rules",
+	"Scam",
+	"Security",
+	"Security Analytics",
+	"Simulation",
+	"Spam",
+	"TIM",
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:33]: We can add `Feed`

File: Tests/Marketplace/approved_tags.json (hunk 2 above)
Comment (glicht) [reply Tests/Marketplace/approved_tags.json:11]: Let's remove them

File: Tests/Marketplace/approved_tags.json [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+	"Dark Web",
+	"Email",
+	"Failed Incidents",
+	"Failed Integrations",
+	"Forensics",
+	"GDPR",
+	"HIPAA",
+	"IAM",
+	"Incident Handling",
+	"Incident Response",
```
Comment (glicht) [inline Tests/Marketplace/approved_tags.json:17]: @michalgold do we need both incident response and incident handling?

File: Tests/Marketplace/approved_tags.json (hunk 3 above)
Comment (glicht) [reply Tests/Marketplace/approved_tags.json:21]: Maybe: Incident Mirroring?

File: Tests/Marketplace/approved_tags.json (hunk 4 above)
Comment (glicht) [reply Tests/Marketplace/approved_tags.json:27]: Agree. Let's remove

File: Tests/Marketplace/approved_usecases.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
+[
+	"Access",
+	"Breach Notification",
+	"Breach and Attack Simulation",
+	"Brute Force",
+	"Case Management",
+	"Compliance",
+	"Crisis Management",
+	"Default",
```
Comment (glicht) [inline Tests/Marketplace/approved_usecases.json:9]: I think we can remove this

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -0,0 +1,2 @@
+/Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
+/Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
```
Comment (glicht) [inline .github/CODEOWNERS:2]: duplicate line
```suggestion
/Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
```

**Comment by glicht** [comment]
@Itay4 for the approved json files let's use a format of:
```json
{
    "description": "Approved tags for use in pack_metadata.json. To modify this file you will need a CODEOWNER to approve.",
    "approved_list": [
          ....
     ]
}
```
This allows us to add a description and in the future any other data we want in a flexible way.

File: Tests/Marketplace/approved_tags.json (hunk 3 above)
Comment (anara123) [reply Tests/Marketplace/approved_tags.json:21]: Incident Mirroring sounds better

File: Tests/Marketplace/approved_tags.json [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+	"Risk",
+	"Rules",
+	"Scam",
+	"Security",
+	"Security Analytics",
+	"Simulation",
+	"Spam",
+	"TIM",
+	"Threat Intelligence",
+	"VPN"
```
Comment (anara123) [inline Tests/Marketplace/approved_tags.json:35]: also `Palo Alto Networks` @glicht ?
For Palo Alto Networks products

File: Tests/Marketplace/approved_usecases.json (hunk 9 above)
Comment (michalgold) [reply Tests/Marketplace/approved_usecases.json:9]: Added on purpose.  I wanted it to be easily found.

File: Tests/Marketplace/approved_usecases.json (hunk 6 above)
Comment (michalgold) [reply Tests/Marketplace/approved_usecases.json:16]: The use case is "Identity And Access Management"

File: Tests/Marketplace/approved_tags.json (hunk 7 above)
Comment (michalgold) [reply Tests/Marketplace/approved_tags.json:33]: No need, we have TIM

File: Tests/Marketplace/approved_tags.json (hunk 5 above)
Comment (Itay4) [reply Tests/Marketplace/approved_tags.json:34]: isnt a category?

**Review by glicht — APPROVED** [review]

**Pull request merged**